README.md

```diff
@@ -1,11 +1,154 @@
+
+From the directory where your *.csproj file is located, run `dotnet pack` and you'll get an output like this:
+
+```
+P SD:\corefx-testdata\System.IO.Compression.TestData> dotnet pack
+
+    Microsoft (R) Build Engine version 16.1.68-preview+g64a5b6be6d for .NET Core
+    Copyright (C) Microsoft Corporation. All rights reserved.
+
+    Restore completed in 28.78 ms for D:\corefx-testdata\System.IO.Compression.TestData\System.IO.Compression.TestData.csproj.
+    C:\Program Files\dotnet\sdk\2.1.700-preview-009618\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.RuntimeIdentifierInference.targets(143,5): message NETSDK1057: You are working with a preview version of the .NET Core SDK. You can define the SDK version via a global.json file in the current project. More at https://go.microsoft.com/fwlink/?linkid=869452 [D:\corefx-testdata\System.IO.Compression.TestData\System.IO.Compression.TestData.csproj]
+    System.IO.Compression.TestData -> D:\corefx-testdata\System.IO.Compression.TestData\bin\Debug\netstandard2.0\System.IO.Compression.TestData.dll
+```
+
+*Note: Running `dotnet pack` from the root of the `corefx-testdata` project will generate the nuget packages for all the \*.csproj files.*
+
+The generated nuget file(s) will be located in the `bin\Debug` subfolder:
+
+`D:\corefx-testdata\System.IO.Compression.TestData\bin\Debug\System.IO.Compression.TestData.1.0.10.nupkg`
+
+
+#### 3. Test that corefx can import the new nuget package file.
+
+We need to make sure our change works by importing the generated nuget package file into our corefx project.
+
+Open the `corefx\external\test-runtime\XUnit.Runtime.depproj` file, find the `<PackageReference>` element for your TestData project and change the version to the same number you used in step 1.
+
+```xml
+<!-- Test Data -->
+<PackageReference Include="System.IO.Compression.TestData" Version="1.0.10" />
+```
+
+Now from the root of the `corefx` project, run the commands that will import the package file:
+
+```
+PS D:\corefx> .\.dotnet\dotnet.exe restore .\external\test-runtime\XUnit.Runtime.depproj -s D:\corefx-testdata\System.IO.Compression.TestData\bin\Debug\
+    Restore completed in 1.08 sec for D:\corefx\external\test-runtime\XUnit.Runtime.depproj.
+
+PS D:\corefx> .\.dotnet\dotnet.exe msbuild .\external\test-runtime\XUnit.Runtime.depproj /t:rebuild
+
+    Microsoft (R) Build Engine version 16.1.67-preview+g13843078ee for .NET Core
+    Copyright (C) Microsoft Corporation. All rights reserved.
+
+    Restore completed in 292.24 ms for D:\corefx\external\test-runtime\XUnit.Runtime.depproj.
+
+     C:\Users\username\.nuget\packages\...
+     C:\Users\username\.nuget\packages\...
+     ...
+```
+
+ Now check if the nuget package was exported correctly. Go to your current user's nuget packages folder, usually located in `C:\Users\yourusername\.nuget\packages`. Find the folder for your TestData and inside you will find the original dependencies plus the new ones you added:
```

Comment:
NIT: `was exported` -> `was restored`

Reply:
NIT: `C:\Users\yourusername\` -> `%UserProfile%`

Reply:
NIT: `you will find the original dependencies plus the new ones you added:` -> `you should see a new folder with the new version of the package that you produced.`

Reply:
Fixed